### ISSUE TITLE
Expand planetary concepts

### ISSUE THREAD

**Issue by mkelley** [astrothesaurus/UAT#265]
Hello,

I have been reviewing the UAT for planetary science concepts.  I work for the NASA Planetary Data System, specifically the Small Bodies Node at the University of Maryland.  We would like to use the UAT as a way to pick subject keywords for our data sets.  However, there are a lot of planetary concepts missing.  So, this issue is just to introduce myself and provide a heads-up that I will be filing several issues for discussion and revision.  I have about 86 suggested concepts or reorganizations!

Cheers,
Mike

**Comment by katieefrey** [comment]
@mkelley

Hi Mike,

This is great!  We're planning to release another update in December, so that will give us plenty of time for us to review and discuss your suggestions for inclusion in the next version.  I will look forward to seeing your contributions.

Best,
Katie

**Comment by mkelley** [comment]
That sounds good, @katieefrey.  I have a checklist below that I will use to keep track of my proposals:

- [ ] Add Delay-Doppler Imaging
- [ ] Move meteor / meteorite concepts from Interplanetary Physics
- [ ] (Re)move Cometary Studies from Interplanetary Physics
- [ ] Expand Planetary Atmospheres (Structure, Dynamics, Evolution, Chemistry, Composition)
- [ ] "Small solar system bodies" and "Minor planets" are near synonyms
- [ ] Add Planetary Surfaces
- [ ] Add Solid Body Tides
- [ ] Move Satellite Formation
- [ ] Rename Meteoroid Dust Clouds
- [ ] Add Zodiacal Dust Bands
- [ ] Remove Interplanetary Grains
- [ ] Consolidate Trans-Neptunian Objects
- [ ] (Re)move Meteor Streams
- [ ] Add Asteroids : Satellites
- [ ] Expand Comets
- [ ] Add Near-Earth Objects
- [ ] Add Interstellar Objects and ʻOumuamua
- [ ] Add Solar System Dynamics

**Comment by mkelley** [comment]
I guess one question I have up front, when is it appropriate to have concepts in multiple locations?  Understanding this better might change my checklist.

**Comment by katieefrey** [comment]
@mkelley 

It's appropriate whenever it makes sense for the concept... ok that's a vague answer.

The parent child relationship describes a few different situations:

1.  generic relationships (some/all, all cacti (child) are succulents (parent), but not all succulents are cacti)
1.  instance relationships (the child is an example of the parent, Cinderella (child) is an example of a fairy tale (parent))
1.  whole/part relationships (the spinal cord (child) is part of the nervous system(parent))

These relationships needs to hold the way down the chain for each chain, meaning if a concept appears in 2 places and that concept has children, those children should logically flow from the grandparents (etc) along both chains.

Another thing to consider is that the hierarchy structure is most useful for machines, whereas the individual concepts are most useful for people.  Someone looking to tag their paper will probably find the concept they want through a search, not by browsing through the graph of concept.  The computer, however, will use the hierarchy structure to help with suggestions and relationships.

Finally, the UAT does have a "Related concept" relationship, where the concepts in question are not parent/child and they arent siblings, but they are related.  A good example is like the Asymptotic Giant Branch and AGB stars.  The Branch itself is like a concept relating to the evolutionary theory of stars, while the AGB stars are stars classified as being on that branch.  The first concept is found under stellar evolution, while the second is a stellar type.

If you aren't sure you can let me know what you are thinking, I like to pass suggestions by a few people to get confirmation, consensus, and clarity.

**Comment by mkelley** [comment]
Sorry for being out of the discussion for so long.  I've been on parental leave for a big chunk of the fall.

Anyway, with the AAS now publishing a planetary science journal, we should coordinate these changes together with AAS... probably the editorial office for the journal.  I'm back to work in January and will initiate further discussion and work then.

**Comment by katieefrey** [comment]
Congratulations!  I totally understand that a new baby can take over your life, which is as it should be.

I also wasn't able to reply and act on suggestions quickly when they were posted this last year, and I'm working on a development roadmap to give everyone a more concrete idea of the timelines involved with a release.

We did at least manage to have some action on some of your suggestions for this release, which is launching tomorrow.

Looking ahead to the next release I think the planetary science section of the UAT is definitely a good target for revisions, and I would be interested to work with you and the AAS editors on making that happen.

**Comment by katieefrey** [comment]
Just commenting to mention that Near Earth Objects is a UAT concept: http://astrothesaurus.org/uat/1092
I checked it off in your list above, since it does exist.  It is currently found under "Asteroids" which may not be an obvious location, so I'm open to moving it or adding it somewhere else.

**Comment by katieefrey** [comment]
more notes...

"Expand Comets"  possibly addressed by #275 ?

"Zodiacal cloud" (http://astrothesaurus.org/uat/1845) has non preferred term "Zodiacal Dust Bands." If they are synonymous, "Zodiacal clouds" has far more results at ADS so I'm inclined to keep that as the preferred term.

"Planetary surfaces" (http://astrothesaurus.org/uat/2113) exists, but perhaps your checklist item refers to expanding it?

**Comment by katieefrey** [comment]
Lightning added (see #334)

**Comment by mkelley** [comment]
The Zodiacal dust bands are substructure within the Solar System's Zodiacal cloud.  See Sykes and Greenberg 1986 https://doi.org/10.1016/0019-1035(86)90063-1 Reach et al. 1997 https://ui.adsabs.harvard.edu/abs/1997Icar..127..461R/abstract and Nesvorny et al. 2003 https://ui.adsabs.harvard.edu/abs/2003ApJ...591..486N/abstract .  The Reach et al. paper has good images.

**Comment by mkelley** [comment]
Regarding near-Earth objects: NEO generally refers to any asteroid or comet with an orbit near Earth.  We use near-Earth asteroid (NEA) or near-Earth comet (NEC, but less common) when we need to be more specific.

OK, looking at the UAT definition, I see that near-Earth comets and near-Earth asteroids are alternates for near-Earth object.  Being that it includes comets, does it still make sense as a more specific concept for Asteroid?

I see that UAT has the dynamical classes for near-Earth asteroids.  There is also Potentially Hazardous Asteroid (PHA).  The definition is at JPL: https://cneos.jpl.nasa.gov/about/neo_groups.html

**Comment by mkelley** [comment]
I have a few more edits for comets, I'll open a new issue.

**Comment by katieefrey** [comment]
You are right about NEO, it should be moved out from under asteroids.  I'll move it to "small solar system bodies" and give it a related concept for asteroids and comets.

Thanks for the articles about Zodiacal dust bands... I'm adding it as a child of "Zodiacal cloud."  

I believe Nesvorny et al. 2003 https://ui.adsabs.harvard.edu/abs/2003ApJ...591..486N/abstract  is also referring to them as "IRAS dust bands" (dust bands discovered by IRAS), and I'm seeing that phrase in a few other articles as well.  Would that make a useful non-preferred term?

**Comment by katieefrey** [comment]
On a related topic, I was talking to Maria Womack, an editor for the Planetary Science Journal, and she said she wasn't sure what the concept "aperiodic comets" referred to.  I traced the concept back to the original [IAU thesaurus](http://www.mso.anu.edu.au/library/thesaurus/english/APERIODICCOMETS.html) and the [IVOA Thesaurus](http://www.astro.physik.uni-goettingen.de/~hessman/rdf/IVOAT/dict/A.html#aperiodicComet).

We came to the conclusion that this concept was probably better described as "interstellar comets," and indeed the 6 papers that have used the concept so far are talking about Interstellar comets, and usually specifically 'Oumuamua or Borisov.  So I am planning to update the prefer term of this concept to "Interstellar comets" which closely addresses your "interstellar objects."  Would that work for you, or is objects a better descriptor than comets?

**Comment by mkelley** [comment]
IRAS dust bands as a non-preferred term: I agree.  I interpret the Nesvorny et al. use of "IRAS dust bands" to mean that they are specifically referring to the bands as observed by IRAS.  In fact, they point out that two of them were not confirmed in COBE observations.  So, Zodiacal dust band is still the general concept.

aperiodic comets: I don't recall encountering that one before.  It does suggest a comet with a hyperbolic orbit.  That could be an interstellar comet, or a Solar System comet that has been perturbed into a hyperbolic orbit and ejected from the Solar System.

More generally about the use of interstellar object vs comet vs asteroid, I think this is similar to the near-Earth object case.  An individual ISO or NEO may be a comet or an asteroid.  The fact that they are an ISO vs NEO vs something else is based on dynamical criteria.

**Comment by katieefrey** [comment]
[x]"Small solar system bodies" and "Minor planets" are near synonyms

Yea, I see this also in the [Etymological Dictionary of Astronomy and Astrophysics](http://dictionary.obspm.fr/index.php?formSearchTextfield=minor+planet&formSubmit=Search&showAll=1).  I've deprecated the concept "Minor planets" and added it as an alternate label for both SSSBs and Asteroids.

**Comment by BartlettAstro** [comment]
@mkelley, @sjarmak, NASA ADS project scientist for planetary science will also be working on planetary science concepts in the UAT.
While I believe this discussion has been fruitful, may I suggest that we move the concept changes suggested herein that have not been implemented to separate issues and we limit future issues to one concept or a few closely related concepts.
